docs/devel/networking.md

```diff
@@ -99,3 +109,99 @@ As of now, the existent binding mechanisms are:
 - [bridge](#bridge-binding-mechanism)
 - [masquerade](#masquerade-binding-mechanism)
 - [slirp](#slirp-binding-mechanism)
+
+### Bridge binding mechanism
+Using the bridge `BindMechanism` requires a VMI configuration featuring a
+network whose interface type is `bridge` - the yaml file below can be used
+as reference, but please refer to the
+[user guide](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=bridge)
+for more information.
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: default
+          bridge: {}
+  networks:
+  - name: default
+    pod: {} # Stock pod network
```

Comment:
@maiqueb At the beginning of the document it says `od networking is out of the scope of this document;` and yet the example here shows that one is able to configure a bridge on top of a pod network interface. 
I think it requires some clarification.

Also, I'm not sure why are we trying to exclude the pod networking from this document and not simply say that we are able to use the bridge binding with any interface that appears in the pod? I think the bridge+veth is still a valid use-case  for some community members,  even if it doesn't play nicely with live migration. Isn't it?

Reply:
I just mean that CNI is out of scope.I hoped to achieve what you're saying w/ `Kubernetes is responsible to set up pod networking according to its
￼configuration, via CNI. It is safe to assume it will simply plug a configured
network interface into the pod, and through it, the pod connects to the
outside world`.